src/sql/workbench/api/node/extHostBackgroundTaskManagement.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import { IMainContext } from 'vs/workbench/api/node/extHost.protocol';
+import { ExtHostBackgroundTaskManagementShape, SqlMainContext, MainThreadBackgroundTaskManagementShape } from 'sql/workbench/api/node/sqlExtHost.protocol';
+import * as sqlops from 'sqlops';
+import * as vscode from 'vscode';
+import { Emitter } from 'vs/base/common/event';
+
+export enum TaskStatus {
```

Comment:
Doesn’t this same Enum already exist somewhere else? Is it not somewhere that we can access here?

Reply:
we cannot access sqlops enum s from extension host

---
In reply to: [202498160](https://github.com/Microsoft/sqlopsstudio/pull/1929#discussion_r202498160) [](ancestors = 202498160)